### DISCUSSION TITLE
Welcome to PDEBench Discussions!

### DISCUSSION THREAD

**Discussion by falesiani** [pdebench/PDEBench#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hu1ple** [comment]
Dear Dr.Alesiani, I am a Computer Science and Technology student from Nanjing University of Posts and Telecommunications in China. I've been learning Multi-task Learning, and I'm studying  a paper you wrote which named "Efficient and Scalable Multi-task Regression on Massive Number of Tasks". I'm very interested in the algorithm -- CCMTL you  proposed. But unfortunately, I failed to find any codes to implement the algorithm. So I ask you for help here and I also tried to contact you by email. I'm looking forward to your reply .(Here is my email: b20030119@njupt.edu.cn,if you fail to receive my mail)

**Reply by falesiani** [reply]
Dear hu1ple, 

you can have a look to our followup work [Towards Interpretable Multi-Task Learning Using Bilevel Programming](https://arxiv.org/abs/2009.05483) and the associated code [GGMTL](https://github.com/falesiani/ggmt).

Thanks
